js/views/modals/Settings/Advanced/Settings.js

```diff
@@ -55,7 +58,7 @@ export default class Advanced extends baseVw {
     const formData = this.getFormData();
 
     this.settings.set(formData);
-    app.settings.set( this.settings.toJSON( ) );
+    // app.settings.set( this.settings.toJSON( ) );
```

Comment:
We should only be setting the global app.settings model on the 'sync' handler of our local this.settings. That way the global model is only updated when the data is successfully saved on the server.